mgc/dscr/oneSample.py

```diff
@@ -0,0 +1,110 @@
+from sklearn.metrics import euclidean_distances
+from ._utils import _CheckInputs
+import numpy as np
+import random
+from .base import DiscriminabilityTest
+from scipy._lib._util import MapWrapper
+
+
+
+class oneSample(DiscriminabilityTest):
```

Comment:
Use UpperCamelCase for class names [(described here)](https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html)

Reply:
I also don't think `oneSample` is a descriptive name but I don't have better ideas at the moment

Reply:
Probably `DiscrimOneSample` just so that it is less confusing because of the `ksample` module